src/apps/Template/Template.jsx

```diff
@@ -289,9 +289,26 @@ const Template = React.createClass({
     );
   },
 
+  renderCodeEditor() {
```

Comment:
are you sure it's making any difference? `<TemplateCodeEditor>` will contain `<Editor>` with all params the same. I moved it to a separate method just to make main `render()` more readable.